src/graphics/aurora/cursor.cpp

```diff
@@ -112,6 +114,12 @@ void Cursor::load() {
 
 	TXI *txi = new TXI();
 	txi->getFeatures().filter = false;
+	
+	//if we need DeS3TC, we decompress the cursor image
```

Comment:
A space after the `//`, and then start with with uppercase, please.

I.e.
`// If we ...`